FirebaseAuthUI/FUIAuth.m

```diff
@@ -299,7 +325,7 @@ - (void)signInWithEmailHint:(NSString *)emailHint
   [self.auth fetchProvidersForEmail:emailHint completion:^(NSArray<NSString *> *_Nullable providers,
                                                            NSError *_Nullable error) {
     if (error) {
-      completion(nil, error);
+      completion(nil, error, nil);
```

Comment:
Is `completion` guaranteed to be nonnull here?

Reply:
Good catch, fixed.